### PR TITLE
Implemented Certificates and ModalUploadCertificate pages

### PR DESCRIPTION
**Pull request by SteffiAntony**
Jira: https://jsw.ibm.com/browse/PFEBMC-2392
Jira: https://jsw.ibm.com/browse/PFEBMC-2373

### CODE REVIEW

File: src/store/modules/SecurityAndAccess/CertificatesStore.js [hunk 1]
```diff
@@ -151,10 +142,12 @@ const CertificatesStore = {
         if (certificateType.limit === certificateCount) {
           return;
         } else {
-          availableUploadTypes.push(certificateType);
+          availableUploadType.push(certificateType);
         }
       });
-      commit('setAvailableUploadTypes', availableUploadTypes);
+      console.log('availableUploadType:', availableUploadType);
```
Comment (nabilpandiyalayil) [inline src/store/modules/SecurityAndAccess/CertificatesStore.js:148]: Remove console log

File: src/views/SecurityAndAccess/Certificates/Certificates.vue [hunk 2]
```diff
@@ -71,14 +72,14 @@
             }}
           </template>
           <template #cell(validFrom)="{ value }">
-            {{ value | formatDate }}
+            {{ value }}
```
Comment (nabilpandiyalayil) [inline src/views/SecurityAndAccess/Certificates/Certificates.vue:75]: use filters

File: src/views/SecurityAndAccess/Certificates/Certificates.vue [hunk 3]
```diff
@@ -71,14 +72,14 @@
...
           </template>
           <template #cell(validUntil)="{ value }">
             <status-icon
               v-if="getDaysUntilExpired(value) < 31"
               :status="getIconStatus(value)"
             />
-            {{ value | formatDate }}
+            {{ value }}
```
Comment (nabilpandiyalayil) [inline src/views/SecurityAndAccess/Certificates/Certificates.vue:82]: here also

**Review by nabilpandiyalayil — CHANGES_REQUESTED** [review]

File: src/assets/styles/bmc/custom/_forms.scss [hunk 4]
```diff
@@ -171,3 +171,12 @@ div[role='group'] {
   background-color: $gray-900;
   border-color: $gray-900;
 }
+.form-control-file {
+  opacity: 0;
+  height: 0;
+  &:focus + span {
+    box-shadow:
+      inset 0 0 0 3px theme-color('primary'),
```
Comment (Nikhil-Ashoka) [inline src/assets/styles/bmc/custom/_forms.scss:179]: Theme-color is dropped in Bootstrap5

**Comment by Nikhil-Ashoka** [comment]
New PR: https://github.com/ibm-openbmc/webui-vue/pull/295

**Pull request closed without merging**